org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/IDebugSession.java

```diff
@@ -14,6 +14,7 @@
 import java.util.List;
 
 import com.sun.jdi.ThreadReference;
+import com.sun.jdi.request.EventRequestManager;
 
 public interface IDebugSession {
```

Comment:
Besides creating a breakpoint, please also consider how to remove it.
For normal breakpoint, we use createBreakpoint and return IBreakpoint. To remove, we call IBreakpoint.close().